web/src/components/vis/VisPca.vue

```diff
@@ -80,16 +103,16 @@ export default {
       const xrange = minmax(points.map(d => d.x), 0.1);
       const yrange = minmax(points.map(d => d.y), 0.1);
 
-      const x = scaleLinear()
+      const scalex = scaleLinear()
```

Comment:
Shadowing.  x and y are shadowed on 177/178.

Reply:
In the arrow functions for `xMean` and `yMean`? I'm surprised the linter cared about that (since lexically, this isn't really a problem, unless those functions meant to reference the outer `x` and `y`). Stylistically, the `x` and `y` in those arrow functions ought to both be some other, more generic name, but I'm fine with changing the scale variable names to what you've got.